core-primitives/attestation-handler/src/attestation_handler.rs

```diff
@@ -78,11 +78,14 @@ pub const REPORT_SUFFIX: &str = "/sgx/dev/attestation/v4/report";
 
 /// Trait to provide an abstraction to the attestation logic
 pub trait AttestationHandler {
-	/// Generates an encoded remote attestation certificate.
+	/// Generates an encoded remote attestation certificate. Returns certificate DER encoded.
```

Comment:
[Wording]
How about using the same wording as in `Returns the DER encoded certificate and the raw DCAP quote.`?
```suggestion
	/// Generates an encoded remote attestation certificate. Returns DER encoded certificate.
```